python/tests/datasources/test_cdc_hiv.py

```diff
@@ -0,0 +1,130 @@
+from unittest import mock
+from pandas._testing import assert_frame_equal
+from datasources.cdc_hiv import CDCHIVData
+import pandas as pd
+import os
+
+
+THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+TEST_DIR = os.path.join(THIS_DIR, os.pardir, "data",)
+GOLDEN_DIR = 'cdc_hiv/golden_data'
+
+GOLDEN_DATA = {
+    'race_national': os.path.join(TEST_DIR, GOLDEN_DIR, 'race_and_ethnicity_national_output.csv'),
+    'age_national': os.path.join(TEST_DIR, GOLDEN_DIR, 'age_national_output.csv'),
+    'sex_national': os.path.join(TEST_DIR, GOLDEN_DIR, 'sex_national_output.csv'),
+    'race_state': os.path.join(TEST_DIR, GOLDEN_DIR, 'race_and_ethnicity_state_output.csv'),
+}
+
+
+def _load_csv_as_df_from_data_dir(*args, **kwargs):
+    dataset, filename = args
+    df = pd.read_csv(os.path.join(TEST_DIR, dataset, filename),
+                     dtype={'FIPS': str, 'Population': float}, skiprows=9, thousands=',', quoting=1)
+    return df
+
+
+@mock.patch('ingestion.gcs_to_bq_util.load_csv_as_df_from_data_dir', side_effect=_load_csv_as_df_from_data_dir)
+def testGenerateRaceNational(
+    mock_data_dir: mock.MagicMock,
+):
+    datasource = CDCHIVData()
+    df = datasource.generate_breakdown_df(
+        'race_and_ethnicity', 'national')
+    expected_df_race_national = pd.read_csv(
+        GOLDEN_DATA['race_national'], dtype={'state_fips': str})
+
+    assert_frame_equal(df, expected_df_race_national, check_like=True)
+
+
+@mock.patch('ingestion.gcs_to_bq_util.load_csv_as_df_from_data_dir', side_effect=_load_csv_as_df_from_data_dir)
+def testGenerateSexNational(
```

Comment:
this test function name should match what it's testing to avoid confusion later. it looks like this one should be `testGenerateAgeNational` and the next one down should be `...Sex...`